packages/db/src/lib/granule.ts

```diff
@@ -1,9 +1,12 @@
 import Knex from 'knex';
 
 import { PostgresGranule } from '../types/granule';
+import { CollectionPgModel } from '../models/collection';
 import { GranulePgModel } from '../models/granule';
 import { GranulesExecutionsPgModel } from '../models/granules-executions';
 
+const { deconstructCollectionId } = require('../../../api/lib/utils');
```

Comment:
```suggestion
const { deconstructCollectionId } = require('@cumulus/message/Collections');
```

We want to require from the cumulus message package rather than cross linking to the API package

Reply:
It seems like this doesn't exist in the message package? But I can certainly add it there. Originally I was requiring this from the api package, but that led to a dependency loop, so I changed it to match another require in this same package https://github.com/nasa/cumulus/blob/master/packages/db/src/translate/granules.ts#L7

I can update both places though

Reply:
Sorry we must have only added that in the feature branch but yeah if we can do that here too that would be ideal. Thanks!